spec/models/peer_review_question_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PeerReviewQuestion, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Would be nice if there weren't any pending specs. Either remove these or write tests?